### DISCUSSION TITLE
User assigned data is changed and incorrect

### DISCUSSION THREAD

**Discussion by RikkiBC** [nextcloud/all-in-one#3312] [Questions] [answered]
### Steps to reproduce
1. Go to Active Users
2. Change a user's assigned data to "100 GB".

### Expected behavior 
User gets assigned "100 GB".

### Actual behavior 
User gets assigned "115.3 GB"

### Host OS 
VM, Ubuntu 22.04, ARM64 architecture

#### Nextcloud AIO version 
v7.0.0

#### Current channel 
`latest-arm64`

**Comment by szaimen** [comment, marked as answer]
Duplicate of https://github.com/nextcloud/server/issues/38874
